ui/src/storage.js

```diff
@@ -1,5 +1,5 @@
 export const save = (s) => {
-  localStorage.setItem('state', JSON.stringify(s));
+  localStorage.setItem('state', JSON.stringify({...load(), ...s}));
```

Comment:
FYI @morelazers I had to reverse the order of what you initially drew up on our call - this way, loaded parameters are overwritten by those in state if they have the same keys.

Reply:
Makes sense.